web/cobrands/sass/_base.scss

```diff
@@ -619,6 +619,10 @@ small.or:after {
     padding-top: 0.6em;
     padding-bottom: 0.5em;
   }
+
+  &:has(.form-error) {
+    @include flex-direction(column);
```

Comment:
This works nicely on /auth, on the alert page it changes it from `[email] [Submit]` to
```
[error]
[email]
[Submit]
```
if you're happy with that, fine with me